process_lunr.xml

```diff
@@ -13,20 +13,33 @@
   <!--
     Setting up Lunr processing. Sets up ANT variables
   -->
-  <target name="lunr.init">
+  <target name="lunr.init" description="Setting up Lunr processing">
 
     <property
       name="lunr.lib"
       value="${dita.plugin.net.infotexture.dita-bootstrap.lunr.dir}/resource/lunr.js"
     />
 
+    <property name="is.Bootstrap" value="true" />
     <condition property="lunr.support.include">
-      <matches
-        string="${default.language}"
-        pattern="((ar|da|de|es|fi|fr|hi|hu|it|ja|jp|ko|nl|no|pt|ro|ru|sv|ta|th|tr|vi|zh)(\-|$))"
-      />
+      <and>
+        <matches
+          string="${default.language}"
+          pattern="((ar|da|de|es|fi|fr|hi|hu|it|ja|jp|ko|nl|no|pt|ro|ru|sv|ta|th|tr|vi|zh)(\-|$))"
+        />
+        <istrue value="${is.Bootstrap}" />
+      </and>
     </condition>
+  </target>
 
+  <!--
+    Creates a JavaScript file for generating the lunr index
+  -->
+  <target
+    name="lunr.node.script.copy"
+    if="is.Bootstrap"
```

Comment:
See previous comment

Reply:
Dependency of `lunr` target, it is only fired if `is.Bootstrap` has been set within the `lunr.init` target, which means `bootstrap.process.pre` has been fired.